Jenkinsfile

```diff
@@ -112,6 +110,47 @@ pipeline {
                 reportFiles: 'index.html',
                 reportName: "Coverage Report"
             ])
+
+            script{
+              withEnv(["HOME=${env.WORKSPACE}"]) {
+                def RESULT = sh returnStatus: true, script: """
+                  source ${env.LSST_STACK}/loadLSST.bash
+
+                  curl -O ${env.PLANTUML_URL}
+
+                  pip install sphinxcontrib-plantuml
+
+                  cd phosim_utils/
+                  setup -k -r . -t ${env.STACK_VERSION}
+
+                  cd ../ts_wep/
+                  setup -k -r .
+
+                  cd ..
+                  setup -k -r .
+
+                  package-docs build
+
+                  pip install ltd-conveyor
+
+                  ltd upload --product ${env.DOCUMENT_NAME} --git-ref \${GIT_BRANCH} --dir doc/_build/html
+                    """
+
+                if ( RESULT != 0 ) {
+                    unstable("Failed to push documentation.")
+                }
+              }
+            }
```

Comment:
Why you do not want to do this step in the **stages** but in the "*post**? I felt it might be cleaner to do it in the **stages**.

Reply:
yeah... I think the problem is that, if I do it in a stage and a stage before it fails, this will not run. Also, if this is a stage and it fail, the stages bellow it will not run.